wpilibc/src/main/native/cpp/Drive/RobotDriveBase.cpp

```diff
@@ -20,11 +20,18 @@ using namespace frc;
 
 RobotDriveBase::RobotDriveBase() { m_safetyHelper.SetSafetyEnabled(true); }
 
+/**
+ * Change the default value for deadband scaling. The default value is
+ * 0.02. Values smaller then the deadband are set to 0, while values
+ * larger then the deadband are scaled from 0.0 to 1.0. See ApplyDeadband.
```

Comment:
This needs to say `ApplyDeadband()` for Doxygen to automatically link to the function.